shepherd_bms/src/main.cpp

```diff
@@ -18,13 +18,12 @@ WDT_T4<WDT1> wdt;
 AccumulatorData_t *prevAccData = nullptr;
 
 uint32_t bmsFault = FAULTS_CLEAR;
-
-uint16_t overVoltCount = 0;
-uint16_t underVoltCount = 0;
-uint16_t overCurrCount = 0;
 uint16_t chargeOverVolt = 0;
-uint16_t overChgCurrCount = 0;
-uint16_t lowCellCount = 0;
+
+//states for fault timers
+enum f{BEFORE_TIMER_START, FAULT_EVAL};
```

Comment:
just some organizational stuff - i think a bit more descriptive name for the enum than `f` would prob be ideal, and just spreading out each state into its own line for readability (see bms_fault enum in datastructs.h for reference)

Reply:
lmao at first glance I thought it was a python f string for a sec

Reply:
^ echoing @dyldonahue comments